wcfsetup/install/files/lib/system/package/plugin/StylePackageInstallationPlugin.class.php

```diff
@@ -32,7 +32,7 @@ public function install() {
 		$styleEditor = StyleHandler::getInstance()->getStyleByName($this->installation->getPackageName(), false);
 		
 		// import style
-		$style = StyleEditor::import($filename, $this->installation->getPackageID(), $styleEditor);
+		$style = StyleEditor::import($filename, $this->installation->getPackageID(), $styleEditor, !PACKAGE_ID);
```

Comment:
Not sure whether font download should ever run during package installation. For now I'm only excluding it during WCFSetup.